vickev/howdypix
# Thumbnails in the albums

## Description

Adds a thumbnail, the number of photos, the number of albums in the album link.

![image](https://user-images.githubusercontent.com/219680/69843527-e56e5000-1235-11ea-9fcb-05b462c446d8.png)






## How to test?

- Check-out this branch.
- Run the project
- The albums must display these information

## Checklist
